src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5210,4 +5210,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_RefExtensionMustBeValueTypeOrConstrainedToOne" xml:space="preserve">
     <value>The first parameter of a 'ref' extension method '{0}' must be a value type or a generic type constrained to struct.</value>
   </data>
+  <data name="ERR_OutAttrOnInParam" xml:space="preserve">
+    <value>Cannot specify Out attribute on an 'in' parameter.</value>
```

Comment:
Our current error messages are like this:
```
error CS0662: Cannot specify only Out attribute on a ref parameter. Use both In and Out attributes, or neither.
error CS0036: An out parameter cannot have the In attribute
```

Would `An in parameter cannot have the Out attribute` be better, since it matches the equivalent scenario for `[In] out`